content/v2.0/influx-templates/_index.md

```diff
@@ -12,4 +14,32 @@ from dashboards and Telegraf configurations to notifications and alerts in a sin
 Use Influx Templates to quickly set up a fresh instance of InfluxDB, backup your
 dashboard configuration, or share your configuration with the InfluxData community.
 
+## Template manifests
+A template manifest is a file that defines the contents of a template.
```

Comment:
## Templates
An InfluxDB template defines....and supports the following formats:

Reply:
I think the language around "manifests" is important. Our community and marketing teams are using it all their communications about templates.

Reply:
Curious what value the term adds? Is it something other than a template?